example/flutter_chat_app/lib/main.dart

```diff
@@ -34,8 +34,9 @@ class MyHomePage extends StatefulWidget {
 
 class _MyHomePageState extends State<MyHomePage> {
   TopicClient _topicClient = TopicClient(
-      CredentialProvider.fromEnvironmentVariable('MOMENTO_API_KEY'),
-      MobileTopicConfiguration.latest());
+      CredentialProvider.fromString(
+          const String.fromEnvironment('MOMENTO_API_KEY')),
```

Comment:
Isn't this equivalent to the simpler `fromEnvironmentVariable` version?

Reply:
I originally tried `fromEnvironmentVariable` but apparently in Flutter I need to use the `const String.fromEnvironment` constructor to get a non-empty string from the environment variable